commands/Moderation/mute.js

```diff
@@ -0,0 +1,36 @@
+exports.run = async (client, msg, [user]) => {
```

Comment:
`user` is a very confusing variable name for a GuildMember instance. Use member instead.